reggie/configs/data/west_virginia.yaml

```diff
@@ -93,72 +121,245 @@ column_names:
   - POLL_NAME
 
 columns:
-  ID_VOTER: character
-  County_Name: character
-  FIRST NAME: character
-  Mid: character
-  LAST NAME: character
-  Suffix: character
+  ID_VOTER: text
+  County_ID: text
+  County_Name: text
+  FIRST NAME: text
+  Mid: text
+  LAST NAME: text
+  Suffix: text
   DATE OF BIRTH: date
-  SEX: character
-  HOUSE NO: character
-  STREET: character
-  STREET2: character
-  UNIT: character
-  CITY: character
-  STATE: character
-  ZIP: character
-  MAIL HOUSE NO: character
-  MAIL STREET: character
-  MAIL STREET2: character
-  MAIL UNIT: character
-  MAIL CITY: character
-  MAIL STATE: character
-  MAIL ZIP: character
+  SEX: text
+  HOUSE NO: text
+  STREET: text
+  STREET2: text
+  UNIT: text
+  CITY: text
+  STATE: text
+  ZIP: text
+  MAIL HOUSE NO: text
+  MAIL STREET: text
+  MAIL STREET2: text
+  MAIL UNIT: text
+  MAIL CITY: text
+  MAIL STATE: text
+  MAIL ZIP: text
   REGISTRATION DATE: date
-  PartyAffiliation: character
-  Status: character
-  Congressional District: character
-  Senatorial District: character
-  Delegate District: character
-  Magisterial District: character
-  Precinct_Number: character
-  POLL_NAME: character
-
-county_codes:
-  Not_Provided: NP
+  PartyAffiliation: text
+  Status: text
+  Congressional District: text
+  Senatorial District: text
+  Delegate District: text
+  Magisterial District: text
+  Precinct_Number: text
+  POLL_NAME: text
+
+ordered_columns:
+  - ID_VOTER
+  - County_Name
+  - FIRST NAME
+  - Mid
+  - LAST NAME
+  - Suffix
+  - DATE OF BIRTH
+  - SEX
+  - HOUSE NO
+  - STREET
+  - STREET2
+  - UNIT
+  - CITY
+  - STATE
+  - ZIP
+  - MAIL HOUSE NO
+  - MAIL STREET
+  - MAIL STREET2
+  - MAIL UNIT
+  - MAIL CITY
+  - MAIL STATE
+  - MAIL ZIP
+  - REGISTRATION DATE
+  - PartyAffiliation
+  - Status
+  - Congressional District
+  - Senatorial District
+  - Delegate District
+  - Magisterial District
+  - Precinct_Number
+  - POLL_NAME
+
+# Modifications table
+##############################
+primary_key:
+  - pre_date
+  - post_date
+  - ID_VOTER
+  - change_type
+
+base_columns:
+  pre_date: timestamp
+  post_date: timestamp
+  ID_VOTER: text
+  change_type: text
+  pre_value: text
+  post_value: text
+  state: text
 
+# Summary table
+##############################
+summary_columns:
+  - date
+  - state
+  - locale
+  - totals
+  - male
+  - female
+  - ages_18_34
+  - ages_35_49
+  - ages_50_64
+  - ages_65_130
+  - unaffiliated

```

Comment:
This will have to match whatever value you decided to consolidate no party voters to. So above you have "no_party_affiliation: No Party Affiliation". It doesn't matter which you pick, they just need to match.